bin/setup

```diff
@@ -2,26 +2,16 @@
 
 [[ -x bin/deps ]] && source bin/deps
 
-[[ -z ${BashMatic} ]] && { 
+[[ -z ${BashMatic} ]] && {
   echo "Failed to initialize BASH library."
   exit 1
 }
 
 # abort on error
 run::set-all abort-on-error
 
-# install rubocop
-lib::gem::install relaxed-rubocop
+lib::gem::cache-refresh
```

Comment:
Ensure that the gem cache is refreshed so that we install rubocop if it's not there.